.eslintrc

```diff
@@ -18,6 +19,7 @@
     "semi": ["error", "always"],
     "space-before-function-paren": ["error", "never"],
     "jsx-quotes": "off",
-    "comma-dangle": "off"
+    "comma-dangle": "off",
```

Comment:
I think this violates the prettier config. I would suggest having comma-dangle as a violation.

@sallar?

Reply:
@perry-mitchell I think those already existed in `.eslintrc` before prettier era.. I have added "prettier" preset so all these extra overrides should be removed from here including this one. But I think this changed was not made by @snhasani

Reply:
Ah good point! My mistake